docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Minishift failed to start because of ```Error: could not create vmnet interface, permission denied or no entitlement```.
+
+    The problem is likely to be that the xhyve driver is not able to clean up vmnet when a VM is removed.
+
+    Workaround:
+        vmnet.framework decides the IP based on following files:
+
+            /var/db/dhcpd_leases
+            /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist
+
+    Reset `minishift` specific IP database and make sure you remove `minishift` entry section from `dhcpd_leases` file. Finally, reboot your system.
+
```

Comment:
Add 'the' > Reset the `minishift` specific IP database and make sure you remove the `minishift` entry section from the `dhcpd_leases` file.